build.gradle

```diff
@@ -96,12 +96,13 @@ githubRelease {
    body = layout.projectDirectory.file('release/changes.md').asFile.text.trim()
 }
 
-tasks.register('createReleaseTag', CreateGitTag) {
+def createReleaseTag = tasks.register('createReleaseTag', CreateGitTag) {
```

Comment:
let's add a comment here to explain why we added this ordering

Reply:
done, let me know if it's ok